src/Sylius/Bundle/AdminBundle/Command/CreateAdminUserCommand.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\AdminBundle\Command;
+
+use Sylius\Component\Core\Model\AdminUserInterface;
+use Sylius\Component\Resource\Factory\FactoryInterface;
+use Sylius\Component\User\Repository\UserRepositoryInterface;
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Intl\Locales;
+
+#[AsCommand(
+    name: 'sylius:admin-user:create',
+    description: 'Create a new admin user'
+)]
+final class CreateAdminUserCommand extends Command
+{
+    private SymfonyStyle $io;
+
+    public function __construct(
+        private UserRepositoryInterface $adminUserRepository,
+        private FactoryInterface $adminUserFactory,
+    ) {
+        parent::__construct();
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output): void
+    {
+        $this->io = new SymfonyStyle($input, $output);
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        if (!$input->isInteractive()) {
+            $this->io->error('This command should be run interactively.');
+
+            return Command::FAILURE;
+        }
+
+        $this->io->title('Admin user creation');
+
+        $confirm = $this->io->confirm('Do you want to create an admin user ?', true);
+
+        if (!$confirm) {
```

Comment:
This confirm step seems useless, the user started an admin user creation, he obviously want it.

Reply:
IMO it's ok. There are many CLI commands that ask for confirmation, such as running migrations

Reply:
The point is that you can CTRL+C at first question here, while if the migration command do not ask it runs migrations, what could be dangerous.

Reply:
How about adding this confirmation at the end, after all data has been entered, to allow the user to review it? 🤔

Reply:
@Nek- , I've removed the confirmation step, it was indeed impractical.
@vvasiloi , I've implemented your idea, the result is attached in the PR description :)

Reply:
@Rafikooo looks great! Thank you!